test/per_file_coverage.sh

```diff
@@ -14,7 +14,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/http/http3:50.0"
 "source/common/init:96.4"
 "source/common/json:90.6"
-"source/common/network:95.3"
+"source/common/network:95.1"
```

Comment:
Did you mean to drop this lower?

/wait-any

Reply:
Yes I meant to revert the effects of #14472 on this particular directory. I think Alyssa's earlier "ratcheting" PR was a bit aggressive and there's something flaky in this test-tree that affects coverage.

I don't think my opportunistic simplification in transport_socket_options_impl.cc will reliably get us closer to 95.3. I think the main issue in this directory is a bunch of constexpr conditionals that were added recently.

Reply:
OK got it, thanks.